css/homestyle.css

```diff
@@ -0,0 +1,123 @@
+body, html {
```

Comment:
in general incearca sa nu stilizezi pe tag-uri, mai ales pe html

Reply:
Aici l-am lasat asa